packages/UI/src/components/organisms/PositionCard/index.ts

```diff
@@ -0,0 +1,10 @@
+export { default } from './PositionCard';
+
+export interface PositionCardProps {
+  // interaction: boolean;
+  // status: boolean;
+  interaction: 'all-h' | 'all-v' | 'save-only' | 'none';
```

Comment:
@Nouri-Anouar 
We will have to talk to Anna (@annaisaacks) to refine those interaction tokens. all-h and all-v as well as save-only doesn't make alot of sense to me.

Reply:
sounds good.

Reply:
@Nouri-Anouar have you discussed it with Anna?

Reply:
@Enjoy2Live no! we didn't.